templates/pages/management/softwarelicense.html.twig

```diff
@@ -111,13 +111,11 @@
    {% set validity_msg = null %}
    {% if item_type == 'SoftwareLicense' %}
       {% set validity_msg %}
-         {% if item.fields['is_valid'] %}
-            <span class="green">{{ _x('adjective', 'Valid') }}</span>
-         {% elseif not item.fields['is_valid'] and item.fields['allow_overquota'] %}
-            <span class="green">{{ _x('adjective', 'Valid (Over Quota)') }}</span>
-         {% else %}
-            <span class="red">{{ _x('adjective', 'Invalid') }}</span>
-         {% endif %}
+         <div class="d-flex justify-content-around pt-2">
+            {{ include('pages/management/license_progressbar.html.twig', {
+               total: item.fields['number']
+            }) }}
```

Comment:
Please add `with_context: false` and explicitely pass required variables to the embedded template.

```suggestion
            }, with_context: false) }}
```